### PR TITLE
Allow the pattern update script to fetch pull requests

### PR DESCRIPTION
**Pull request by thewilkybarkid**
Should help cases where there's been multiple merges on the Pattern Library and we need incremental PRs here, as well as experimental usage (ie the Pattern Library PR _hasn't_ yet been merged, eg for the likes of #42).

/cc @nlisgo

### CODE REVIEW

File: bin/update-patterns [hunk 1]
```diff
@@ -9,6 +9,24 @@ function finish {
     docker rm browser-pattern-library &> /dev/null || true
 }
 
+if [[ $# -eq 0 ]]
```
Comment (giorgiosironi) [inline bin/update-patterns]: I'd pass this script through `shellcheck`, which will probably make you add a lot of `"` around variables for safety and other suggestions

File: bin/update-patterns [hunk 2]
```diff
@@ -9,6 +9,24 @@ function finish {
...
+  then
```
Comment (giorgiosironi) [inline bin/update-patterns]: `then` usually on the same line as `if`?

File: bin/update-patterns (hunk 1 above)
Comment (thewilkybarkid) [reply bin/update-patterns]: Already done!

File: bin/update-patterns [hunk 3]
```diff
@@ -9,6 +9,21 @@ function finish {
     docker rm browser-pattern-library &> /dev/null || true
 }
 
+if [[ $# -eq 0 ]]; then
+    commit=origin/master
+elif [[ $# -eq 1 && $1 =~ ^pr-([1-9][0-9])*$ ]]; then
+    pr=${BASH_REMATCH[1]}
+elif [[ $# -eq 1 ]]; then
+    commit=$1
+else
```
Comment (nlisgo) [inline bin/update-patterns]: How can I trigger the `else` condition?

Comment (thewilkybarkid) [reply bin/update-patterns]: More than 1 argument.

**Pull request merged**